data/korath/korath ships.txt

```diff
@@ -387,6 +387,172 @@ ship "'olofez"
+		"bunks" 1433
+		"mass" 1660
+		"drag" 27.6
+		"heat dissipation" .36
+		"fuel capacity" 700
+
+		"cargo space" 351
+		"outfit space" 1019
+		"weapon capacity" 371
+		"engine capacity" 309
+		weapon
+			"blast radius" 600
+			"shield damage" 9000
+			"hull damage" 4800
+			"hit force" 20000
+	outfits
+		"Banisher Grav-Turret" 4
+		"Warder Anti-Missile" 4
+		"Langrage Hyper-Heaver" 1
+		"Firelight Missile Bank"
+		"Firelight Missile" 18
+
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Fuel Processor" 2
+		"Korath Repeater Rifle" 523
+		"Microbot Defense Station" 18
+
+		"Afterburner (Planetary Class)"
+		"Reverser (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+		"Jump Drive"
+
+	"reverse engine" -79 -186
+		zoom 0.8
+	"reverse engine" 79 -186
+		zoom 0.8
+	engine -101 196
+		zoom 0.9
+	engine 101 196
+		zoom 0.9
+	engine -60 190 0.8
+	engine 60 190 0.8
+	engine -29 212
+	engine 29 212
+	gun -81.5 58 "Firelight Missile Bank"
+		angle 270
+	turret -89 -134 "Banisher Grav-Turret"
+	turret 89 -134 "Warder Anti-Missile"
+	turret -110 -126 "Warder Anti-Missile"
+	turret 110 -126 "Banisher Grav-Turret"
+	turret 36.5 -55.5 "Langrage Hyper-Heaver"
+	turret -63.5 44 "Warder Anti-Missile"
+	turret 63.5 44 "Banisher Grav-Turret"
+	turret -63.5 66.5 "Banisher Grav-Turret"
+	turret 63.5 66.5 "Warder Anti-Missile"
+	bay "Fighter" -115 -152
+		angle 30
+	bay "Fighter" 115 -152
+		angle 330
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Although fierce in battle and highly adaptable, Korath do not easily recover from injury. Without the aid of a surgeon or more equipment than fits into a standard escape pod, a stranded, injured Korath faces poor odds of survival. Therefore, Korath who leave their home territory have placed great importance on the construction of hospital ships that can quickly locate, recover, and treat the survivors of wrecked ships."
+	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have at hand. Even incubators and nurseries are available to treat the young who grow up aboard ship."
+
+
+ship "Ikatila'ej"
+	sprite "ship/ikatila-ej"
+	thumbnail "thumbnail/ikatila-ej"
+	attributes
+		category "Utility"
+		"cost" 109375000
+		"shields" 72700
+		"hull" 93000
+		"required crew" 957
+		"bunks" 1933
+		"mass" 5380
+		"drag" 58
+		"heat dissipation" .29
+		"fuel capacity" 800
+		"cargo space" 845
+		"outfit space" 1829
+		"weapon capacity" 680
+		"engine capacity" 290
+		"spinal mount" 1
```

Comment:
```suggestion
		"outfit space" 1153
		"weapon capacity" 680
		"engine capacity" 290
		"energy generation" 60
		"heat generation" 95
		"energy capacity" 15600
		"shield generation" 9
		"shield energy" 13.2
		"hull repair rate" 1.2
		"hull energy" 1.2
		"hull heat" 1.8
		"spinal mount" 1
```

Reply:
Ug. Not a fan. Why are we stopping it all the outfits in order to build all the same stuff into the hull?!

Reply:
This ship has a ridiculous amount of outfit space (even by the standards of some other recent ships).
The microbot factory is also supposed to be a significant part of the ship, but as it is, it is essentially a battleship that happens to have a really close range gun on the front.
My alternative suggestion is that all the extra weapons be removed and it just be the factory on the front and the outfits to support that (and a systems core), but I suspect that would not be appreciated.

Why do we not want built in stats? I'm taking off the outfit space those outfits used to occupy, while it still has a similar effective outfit space, it is at least not customisable anymore.

Reply:
We are fine with the Bactrians being sort customizable. And w are equally fine with luxury quarters being used to host boarding canon fodder. Stuff being used for stuff other than what the original designers intended is part of the miracle and curse of highly modular ships. I don't see a reason to start locking down ships to prevent them from being useful to the player as desirable now.